bin/node/testing/Cargo.toml

```diff
@@ -39,7 +39,7 @@ substrate-test-client = { version = "2.0.0-rc6", path = "../../../test-utils/cli
 pallet-timestamp = { version = "2.0.0-rc6", path = "../../../frame/timestamp" }
 pallet-transaction-payment = { version = "2.0.0-rc6", path = "../../../frame/transaction-payment" }
 pallet-treasury = { version = "2.0.0-rc6", path = "../../../frame/treasury" }
-wabt = "0.9.1"
+wat = "1.0"
```

Comment:
Is this needed at all? Wondering because we don't have any other changes to this crate in this PR ... sounds like it is not needed?

Reply:
Good catch! Let's run an experiment